core/sdk/src/api/observability/operational_metrics.rs

```diff
@@ -1,9 +1,220 @@
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+
 use crate::{
-    api::firebolt::fb_telemetry::OperationalMetricRequest,
     extn::extn_client_message::{ExtnPayload, ExtnPayloadProvider, ExtnRequest},
     framework::ripple_contract::RippleContract,
 };
 
+/*
+Operational Metrics. These are metrics that are not directly related to the user's behavior. They are
+more related to the operation of the platform itself. These metrics are not sent to the BI system, and
+are only used for operational/performance measurement - timers, counters, etc -all of low cardinality.
+*/
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+pub struct Counter {
```

Comment:
This (and Timer, etc.) seem almost the same as those defined in `core/sdk/src/api/observability/mod.rs`?

Reply:
thanks , will clean up

Reply:
cleaned